Assets/Scripts/Rendering/VideoRecorderUtils.cs

```diff
@@ -153,7 +154,7 @@ static public class VideoRecorderUtils {
                   Path.ChangeExtension(filePath, "usda") : null;
```

Comment:
This is where the path variable is being set, and where the null is probably coming from.